### PR TITLE
2015 06 mn add badges to kiezkassen

### PR DESCRIPTION
**Pull request by MagdaN**

### CODE REVIEW

**Comment by pallix** [comment]
I'm done with gardening for the backend side. 

@xi can you do the review of the frontend part?

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts [hunk 1]
```diff
@@ -110,8 +143,10 @@ var bindPath = (
                                     commentCount: poolSheet.count,
                                     lng: pointSheet.coordinates[0],
                                     lat: pointSheet.coordinates[1],
-                                    polygon: polygon
+                                    polygon: polygon,
+                                    assignments: resourceAssignments
                                 };
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts]: indentation is off.

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts [hunk 2]
```diff
@@ -96,7 +127,9 @@ var bindPath = (
                             var locationUrl = process.data[SILocationReference.nick]["location"];
                             adhHttp.get(locationUrl).then((location) => {
                                 var polygon = location.data[SIMultiPolygon.nick]["coordinates"][0][0];
-                                scope.data = {
+                                getBadges(adhHttp, $q)(resource).then((assignments) => {
+                                    var resourceAssignments = assignments;
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts]: why this renaming?

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts [hunk 3]
```diff
@@ -47,11 +52,36 @@ export interface IScope extends angular.IScope {
...
     resource: RIProposalVersion;
 }
 
+export class BadgeAssignment {
+    constructor(
+        private title : string,
+        private description : string,
+        private name : string
+    ) {}
+}
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts]: Why is this a class and not just an interface? It feels like overkill.

Comment (MagdaN) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts]: I did like in mercator

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts (hunk 2 above)
Comment (MagdaN) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Kiezkassen/Proposal/Proposal.ts]: yeah, probably could skip that and add assignments to scope.data right away

**Comment by xi** [comment]
LGTM.

**Pull request merged**